src/utilities/theme/types.ts

```diff
@@ -1,3 +1,5 @@
+import {Config} from '@shopify/polaris-tokens/dist-modern/types';
```

Comment:
```suggestion
import {Config} from '@shopify/polaris-tokens/dist-modern';
```

Reply:
Unfortunately types are not exported from the index

Reply:
Rats. We should fix that though if this type is considered part of the public API